portfolio/src/main/webapp/script.js

```diff
@@ -19,21 +19,50 @@ const RIGHT_ANSWERS = ["choice-2", "choice-2", "choice-1"];
 const MOVIES =
       ['Austin Powers', 'Ace Ventura', 'Talladega Nights', 'The dictator'];
 
-const QUESTION_2 = {
-    question: "Which soccer team do I support?",
-    choice1: "Manchester City",
-    choice2: "Liverpool"
-}
+const QUIZ = document.getElementById("quiz");
+
+/**
+ *  A class to organize questions for the quiz section
+ */
+class Question{
+
+    /**
+     * @param {string} prompt The question being asked
+     * @param {string} choice1 The first choice
+     * @param {string} choice2 The second choice
+     */
+    constructor(prompt, choice1, choice2){
+        this.prompt_ = prompt;
+        this.choice1_ = choice1;
+        this.choice2_ = choice2;
+    }
+
+    /** @return {string} */
+    getPrompt(){
+        return this.prompt_;
+    }
+
+    /** @return {string} */
+    getChoice1(){
+        return this.choice1_;
+    }
 
-const QUESTION_3 = {
-    question: "Which of the following apps don't I have on my phone?",
-    choice1: "Tiktok",
-    choice2: "Reddit"
+    /** @return {string} */
+    getChoice2(){
+        return this.choice2_;
+    }
 }
 
-const QUESTIONS_LIST = [QUESTION_2,QUESTION_3];
+const question2 = new Question("Which soccer team do I support?", 
```

Comment:
the style guidlines suggest that const objects can be cammelCased but I didn't find anything explicit, so just wanted to make sure this is okay?

Reply:
Yep nice catch!
Check out the definition of a constant here: https://google.github.io/styleguide/jsguide.html#naming-constant-names

Essentially, you should only use CONSTANT_CASE if the variable is deeply immutable, objects will rarely fit this criteria, unless you use Object.freeze()